### DISCUSSION TITLE
Flutter: Error: Field 'wLanguage' cannot be nullable or have type 'Null', it must be `int`, `double`, `Pointer`, or a subtype of `Struct` or `Union`.

### DISCUSSION THREAD

**Discussion by mglenna** [codemagic-ci-cd/codemagic-docs#1701] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

When building the IOS app for distribution on the AppStore I get the following error code: 

![error](https://github.com/codemagic-ci-cd/codemagic-docs/assets/44913096/0736fcbb-70ad-4045-a697-a28e6501973e)

### Expected results

I had the error code earlier when compiling locally, but this got fixed when I used the suggested approach:

![discussion](https://github.com/codemagic-ci-cd/codemagic-docs/assets/44913096/eba71f55-5196-47f1-8d5d-1d7bf3cd0ffe)

### Actual results

This approach does not however work when using the build machine as one has to be able to edit the file_version_info.dart on the build machine itself.

### Build id (optional)

646ca3d14d681b0b29221840

**Comment by icarusdust** [comment]
Hi @mglenna, if the issue is related to `file_version_info.dart`, then after making that change in the project, try cleaning caches. I see that you have already resolved the issue anyways. Let me know if otherwise.
